Frontend/ui-driver/src/Screens/ReportIssueChatScreen/Controller.purs

```diff
@@ -327,7 +327,7 @@ eval (RecordAudioModelAction RecordAudioModel.OnClickStop) state =
 
 eval (RecordAudioModelAction RecordAudioModel.OnClickDone) state =
   continueWithCmd state { data { recordAudioState { isUploading = true } } } [do
-    _ <- pure $ startLottieProcess "audio_upload_animation" (getNewIDWithTag "audio_recording_done") true 1.0 "FIT_CENTER"
+    _ <- pure $ startLottieProcess animationConfig{ rawJson = "audio_upload_animation", lottieId = (getNewIDWithTag "audio_recording_done"), scaleType = "FIT_CENTER" }
```

Comment:
@ashishsingh101 can we put these configs inside a function and call that? instead of directly passing it from here.

Reply:
That would not be an optimal approach, since we have to pass something for that argument